erpnext/support/doctype/issue/issue.js

```diff
@@ -4,16 +4,45 @@ frappe.ui.form.on("Issue", {
 	},
 
 	"refresh": function(frm) {
-		if(frm.doc.status==="Open") {
-			frm.add_custom_button(__("Close"), function() {
-				frm.set_value("status", "Closed");
-				frm.save();
-			}, __("Status"));
-		} else {
-			frm.add_custom_button(__("Reopen"), function() {
-				frm.set_value("status", "Open");
-				frm.save();
-			}, __("Status"));
-		}
-	}
+        last_status = frm.doc.status;
+	},
+    "status": function(frm) {dialog(frm)}
 });
+function dialog(frm){
+    var d = new frappe.ui.Dialog({
+                    title: __("Details for "+ frm.doc.status),
+                    fields: [
+                        {
+                            "fieldtype": "Text",
+                            "label": __("Resolution Details"),
+                            "reqd": 1,
+							"name":"resolution_details"
+                        },
+                        {
```

Comment:
set `primary_action` instead of a button. The current UI pattern is for the submit button on the toolbar not on the form!